src/main/java/com/sequenceiq/cloudbreak/service/stack/connector/aws/CloudFormationStackStatusChecker.java

```diff
@@ -0,0 +1,68 @@
+package com.sequenceiq.cloudbreak.service.stack.connector.aws;
+
+import java.util.List;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.amazonaws.AmazonServiceException;
+import com.amazonaws.services.cloudformation.AmazonCloudFormationClient;
+import com.amazonaws.services.cloudformation.model.DescribeStacksRequest;
+import com.amazonaws.services.cloudformation.model.DescribeStacksResult;
+import com.amazonaws.services.cloudformation.model.StackStatus;
+import com.sequenceiq.cloudbreak.domain.Resource;
+import com.sequenceiq.cloudbreak.domain.ResourceType;
+import com.sequenceiq.cloudbreak.domain.Stack;
+import com.sequenceiq.cloudbreak.logger.MDCBuilder;
+import com.sequenceiq.cloudbreak.service.StatusCheckerTask;
+
+public class CloudFormationStackStatusChecker implements StatusCheckerTask<CloudFormationStackPollerContext> {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(CloudFormationStackStatusChecker.class);
+
+    @Override
+    public boolean checkStatus(CloudFormationStackPollerContext context) {
+        boolean result = false;
+        Stack stack = context.getStack();
+        StackStatus successStatus = context.getSuccessStatus();
+        AmazonCloudFormationClient client = context.getCloudFormationClient();
+        Resource resource = stack.getResourceByType(ResourceType.CLOUDFORMATION_STACK);
+        String cloudFormationStackName = resource.getResourceName();
+        DescribeStacksRequest describeStacksRequest = new DescribeStacksRequest().withStackName(cloudFormationStackName);
+        try {
+            MDCBuilder.buildMdcContext(stack);
+            LOGGER.info("Checking AWS CloudFormation stack status of: {} to reach: {}", cloudFormationStackName, successStatus);
```

Comment:
Please change this log msg to: Checking if AWS CloudFormation stack '{}' reached status '{}'